### DISCUSSION TITLE
Handling of the static files of the OEO for the OEO-Viewer

### DISCUSSION THREAD

**Discussion by wingechr** [openego/oeplatform#1025] [General]
@adelmemariani the static files for oeo_viewer are being generated by parcel. should they be committed (like other django apps) or not?

**Comment by jh-RLI** [comment]
As far as I can see, I'm all for committing them. I don't really like those files that much, but in practice it's easier to deploy the changes if they are already available on Git Pull. Also, it's easier for the developers as they don't have to build first and see the changes immediately.

But it might be better to build the js and css files using the tools available on the current system.

**Comment by adelmemariani** [comment]
The static files can be generated with `npm run build`, technically there's no need to push them on GitHub, but it makes the deploy process easier.
